### PR TITLE
Add doubly linked list with javascript

### PR DESCRIPTION
**Pull request by brunocalou**
**Changes:**
This commit adds a javascript implementation of a doubly linked list

### CODE REVIEW

File: code/data_structures/linked_list/doubly_linked_list/doubly_linked_list.js [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+ */
+LinkedList.prototype.get = function (index) {
+
+    if (index < 0) {
+        return undefined;
+    }
+
+    var element;
+
+    this.every(function (elem, i) {
```
Comment (arnavb) [inline code/data_structures/linked_list/doubly_linked_list/doubly_linked_list.js]: A for loop would be clarify the intent more of this function (`get`), and shorten it.

File: code/data_structures/linked_list/doubly_linked_list/doubly_linked_list.js [hunk 2]
```diff
@@ -0,0 +1,516 @@
...
+LinkedList.prototype.getFirst = function () {
+    return this.get(0);
+};
+
+/**
+ * Returns the last element in the list
+ * @returns {any} The last element in the list
+ */
+LinkedList.prototype.getLast = function () {
+    return this.get(this._length - 1);
```
Comment (arnavb) [inline code/data_structures/linked_list/doubly_linked_list/doubly_linked_list.js]: I would keep track of the last element.

File: code/data_structures/linked_list/doubly_linked_list/doubly_linked_list.js [hunk 3]
```diff
@@ -0,0 +1,516 @@
...
+/**
+ * Returns the index of the element in the list
+ * @param {any} element - The element to search
+ * @returns {number} The index of the element in the list, or -1 if not found
+ */
+LinkedList.prototype.indexOf = function (element) {
+
+    var index = -1;
+
+    this.every(function (elem, i) {
```
Comment (arnavb) [inline code/data_structures/linked_list/doubly_linked_list/doubly_linked_list.js]: Same comment here as the one for `get`

File: code/data_structures/linked_list/doubly_linked_list/doubly_linked_list.js [hunk 4]
```diff
@@ -0,0 +1,516 @@
...
+
+    return index;
+};
+
+/**
+ * Returns if the list is empty
+ * @returns {boolean} If the list if empty
+ */
+LinkedList.prototype.isEmpty = function () {
+    return !this._length;
```
Comment (arnavb) [inline code/data_structures/linked_list/doubly_linked_list/doubly_linked_list.js]: Clearer to say: `this._length === 0;`

File: code/data_structures/linked_list/doubly_linked_list/doubly_linked_list.js [hunk 5]
```diff
@@ -0,0 +1,516 @@
...
+ */
+LinkedList.prototype.get = function (index) {
+
+    if (index < 0 || index >= this._length) {
+        return undefined;
+    }
+    
+    var node = this._front;
+
+    for (var i = 0, length = this._length; i < length; i += 1) {
```
Comment (arnavb) [inline code/data_structures/linked_list/doubly_linked_list/doubly_linked_list.js]: This `length` variable is unnecessary.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**